zipkin-lens/index.html

```diff
@@ -16,12 +16,12 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <base href="/zipkin/">
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
I guess you are removing this because there is nothing in vite like PUBLIC_URL in react-scripts?

Reply:
Yes exactly. The base path for vite needs to be set in the vite config ts. Then all paths are prepended with the correct url.